src/components/common/productCard/productCard.module.css

```diff
@@ -139,6 +139,7 @@
   }
 
   .imgWrapper img {
-    width: 343px;
+    width: 100%;
```

Comment:
@IhorMasechko, if you set the width to 100%, the image size for the mobile version will be incorrect compared to how it is drawn in the design in Figma, see the attached screenshot. You can use both `max-width` and `min-width` property for this case. Please contact me if you need any clarification.

![wrong-product-image-size-mobile](https://github.com/user-attachments/assets/6a3b1168-e579-4814-a39a-86c5a21fcad4)

Reply:
I have reviewed the fixes and it looks OK now! Good job!